index.d.ts

```diff
@@ -44,6 +44,21 @@ export type Instance = {
 
 export type Unmount = () => void;
 
+/**
+ * Hook that calls inputHandler callback with input that program received.
+ * Additionally contains helpful metadata for detecting when arrow keys were pressed.
+ */
```

Comment:
Include a usage example like in the readme. I would recommend just using all the text from the readme here. It makes it much easier to keep both in sync.